app/static/pages/home/Home.js

```diff
@@ -13,11 +13,74 @@
 // limitations under the License.
 
 oppiaFoundationWebsite.controller('HomePage', [
-  'WindowDimensionsService', '$scope', function(
-      WindowDimensionsService, $scope) {
-    $scope.desktopView = WindowDimensionsService.isDesktopViewWidth();
-    WindowDimensionsService.registerOnResizeHook(function() {
-      $scope.desktopView = WindowDimensionsService.isDesktopViewWidth();
-      $scope.$apply();
-    });
+  'ADMIN_EMAIL_ADDRESS', 'LearnerProfilesService', '$scope', '$window',
+  function(
+      ADMIN_EMAIL_ADDRESS, LearnerProfilesService, $scope, $window) {
+    $scope.ADMIN_EMAIL = ADMIN_EMAIL_ADDRESS;
+    $scope.QUESTION_EMAIL_SUBJECT = 'Question%20about%20Oppia';
+    $scope.activeTabIndex = $window.sessionStorage.getItem('activeTabIndex');
+    if ($scope.activeTabIndex === null) {
+      $scope.activeTabIndex = 0;
+    }
+    $scope.onTabSelected = function(activeTabIndex) {
+      $scope.activeTabIndex = activeTabIndex;
+      $window.sessionStorage.setItem(
+        'activeTabIndex', activeTabIndex.toString());
+    };
+    $scope.engagementDescriptionText = {
+      donate: 'When you donate to Oppia, you empower a movement towards ' +
+      'better learning for everyone. Every little bit counts!',
```

Comment:
Indent all continuation lines by 2, ditto below.

Reply:
Done here and in other files as well!